selfdrive/car/docs_definitions.py

```diff
@@ -165,10 +185,12 @@ def init(self, CP: car.CarParams, all_footnotes: Dict[Enum, int]):
       self.min_enable_speed = CP.minEnableSpeed
 
     # harness column
-    harness_col = self.harness.value
-    if self.harness is not Harness.none:
+    harness_col = self.harness_kit.connector.value
+    if self.harness_kit.connector is not Harness.none:
       model_years = self.model + (' ' + self.years if self.years else '')
       harness_col = f'<a href="https://comma.ai/shop/comma-three.html?make={self.make}&model={model_years}">{harness_col}</a>'
+      harness_col = f'<img width=2000><details><summary>Content</summary><br>{harness_col}<br><sub>' + '<br><sub>'.join([f"{self.harness_kit.parts.count(part)} {part.value}</sub>" for part in set(self.harness_kit.parts)]) + '</details> &nbsp;'
```

Comment:
I see the problem with the width now. It might be better to put the img tag in the column title once so there's not invisible links that are easy to press when expanding the content.